testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt

```diff
@@ -89,11 +92,32 @@ class MiniAppInputActivity : MenuBaseActivity(), PreloadMiniAppWindow.PreloadMin
     }
 
     private fun displayMiniApp() = when(display) {
-        is InputDisplay.AppId -> preloadMiniAppWindow.initiate(null, display.input.trim(), this)
+        is InputDisplay.AppId -> {
+            initiatePreloadScreen(display.input.trim())
+        }
```

Comment:
You can remove the bracket here to have a single line.

Reply:
Yeah, removed the bracket.